### PR TITLE
webUI - Get user from api when not in cache

### PR DESCRIPTION
**Pull request by guy-har**
Currently:
-  The user information is saved in the local cache on login.
-  The login token is saved in cookie
- user navbar (which includes logout) is shown only if the is user information in the cache

In case the local cache was erased for some reason, users will not have an option to logout 

This PR fixes this partly 
- we will see the user in the navbar and have an option to logout but the value under `user:` will not be available

### CODE REVIEW

**Review by nopcoder — CHANGES_REQUESTED** [review]

**Review by johnnyaug — CHANGES_REQUESTED** [review]
I don't understand this change:
1. When logging in - why would I every want to use the cache? This may be dangerous.
2. The other use of the getCurrentUser function is in lib/hooks/users.jsx and it doesn't pass the accessKeyId parameter.

File: webui/src/lib/components/navbar.jsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const NavUserInfo = () => {
         <Navbar.Collapse className="justify-content-end">
             <NavDropdown title={user.friendly_name || user.id} className="navbar-username" alignRight>
                 <NavDropdown.Header>
-                    User: <code>{user.accessKeyId}</code>
+                    User: <code>{user.accessKeyId || user.id}</code>
```
Comment (johnnyaug) [inline webui/src/lib/components/navbar.jsx:23]: So this will sometimes show the user ID and sometime the access key? Better just not show anything.

**Review by johnnyaug — APPROVED** [review]
Suggestion to hide access key if it's not there.

File: webui/src/lib/components/navbar.jsx (hunk 1 above)
Comment (guy-har) [reply webui/src/lib/components/navbar.jsx:23]: @johnnyaug By not showing anything you mean Not showing anything when there is no accessKey or Not showing anything at all?

Comment (johnnyaug) [reply webui/src/lib/components/navbar.jsx:23]: I say remove it altogether, otherwise strange behaviors can be expected.

**Comment by arielshaqed** [comment]
Found a useful reference for the underlying issue with all technical details included: https://shouldiblamecaching.com/

**Pull request merged**